src/routes/static.ts

```diff
@@ -1,19 +1,22 @@
 // Copyright © 2022 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
-import {
-  defaultLightTheme,
-  RegisterOryElementsExpress,
-} from "@ory/elements-markup"
+import { defaultLightTheme, oryTheme } from "@ory/elements-markup"
+import { assignInlineVars } from "@vanilla-extract/dynamic"
 import express from "express"
-import { defaultConfig, RouteRegistrator } from "../pkg"
+import { RouteRegistrator } from "../pkg"
 import sdk from "../pkg/sdk"
 
-export const registerStaticRoutes: RouteRegistrator = (
-  app,
-  createHelpers = defaultConfig,
-) => {
-  RegisterOryElementsExpress(app, defaultLightTheme, createHelpers)
+export const registerStaticRoutes: RouteRegistrator = (app) => {
   app.use("/", express.static("public"))
+  app.use("/theme.css", (req, res) => {
+    res.header("Content-Type", "text/css")
+    res.send(
+      `body {${assignInlineVars(oryTheme, {
+        ...oryTheme,
+        ...defaultLightTheme,
+      }).toString()}}`,
+    )
+  })
   app.use("/.well-known/ory/webauthn.js", (req, res) => {
     res.contentType("text/javascript")
```

Comment:
Why move it from Ory Elements?

https://github.com/ory/elements/blob/main/src/markup/express.ts